Core/src/main/java/com/plotsquared/core/plot/flag/implementations/TitleFlag.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *       _____  _       _    _____                                _
+ *      |  __ \| |     | |  / ____|                              | |
+ *      | |__) | | ___ | |_| (___   __ _ _   _  __ _ _ __ ___  __| |
+ *      |  ___/| |/ _ \| __|\___ \ / _` | | | |/ _` | '__/ _ \/ _` |
+ *      | |    | | (_) | |_ ____) | (_| | |_| | (_| | | |  __/ (_| |
+ *      |_|    |_|\___/ \__|_____/ \__, |\__,_|\__,_|_|  \___|\__,_|
+ *                                    | |
+ *                                    |_|
+ *            PlotSquared plot management system for Minecraft
+ *                  Copyright (C) 2021 IntellectualSites
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.plotsquared.core.plot.flag.implementations;
+
+import com.plotsquared.core.configuration.caption.TranslatableCaption;
+import com.plotsquared.core.plot.flag.FlagParseException;
+import com.plotsquared.core.plot.flag.PlotFlag;
+import com.plotsquared.core.util.ItemUtil;
+import com.sk89q.worldedit.world.item.ItemType;
+import com.sk89q.worldedit.world.item.ItemTypes;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+public class TitleFlag extends PlotFlag<String[], TitleFlag> {
+
+    public static final TitleFlag TITLE_FLAG_EMPTY = new TitleFlag(new String[0]);
+
+    /**
+     * Construct a new flag instance.
+     *
+     * @param value Flag value
+     */
+    protected TitleFlag(String[] value) {
+        super(value, TranslatableCaption.of("flags.flag_category_string"), TranslatableCaption.of("flags.flag_description_title"));
+    }
+
+    @Override
+    public TitleFlag parse(@NonNull String input) throws FlagParseException {
```

Comment:
This should probably accept something like `/plot flag set Hello World`, similar to the greeting flag too.

Reply:
Would that therefore default to only the title being "hello world"? Or should we assume the first "word" is the title, and the second, the subtitle?

Reply:
I'd expect it to be the title only then. Not sure what the others think about it